lms/djangoapps/teams/static/teams/js/spec/views/team_profile_header_actions_spec.js

```diff
@@ -0,0 +1,254 @@
+define([
```

Comment:
Can you move files instead of deleting/adding? That makes it possible to see what actually changed (and keeps file history).